vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -195,12 +196,19 @@ public VimeoAccount getVimeoAccount() {
 
     public void setVimeoAccount(@Nullable VimeoAccount vimeoAccount) {
         if (vimeoAccount == null) {
-            vimeoAccount = new VimeoAccount();
+            vimeoAccount = new VimeoAccount(this.configuration.accessToken);
+            this.configuration.saveAccount(vimeoAccount, null, null);
```

Comment:
Let's check that accessToken is not null - if it is, we should fallback to the previous way the vimeoAccount was created (default constructor)